examples/meta-macro.kl

```diff
@@ -0,0 +1,30 @@
+#lang "n-ary-app.kl"
```

Comment:
I wrote these while trying to hunt down a bug which said

```
Internal error during expansion! This is a bug in the implementation.
Current context won't accept expressions
```

It turned out the bug had nothing to do with the complexities of macro-defining macros... you simply can't have a raw `(lambda (x) x)` at the top-level, you must wrap it in a `define`, `define-macros`, or `example` 😅

Reply:
Yeah, that needs a proper error constructor rather than just using `InternalError`.

Reply:
This got fixed in the type checker branch, and is now merged.